src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import nimbleLogoWhite from 'assets/images/icons/nimble-logo-white.svg';
```

Comment:
I think we are going to import a lot of assets in the coming days:
what about we set the path in `tsconfig.json` so that we can import it like this:

import nimbleLogoWhite from 'icons/nimble-logo-white.svg';

```json
"paths": {
        "icons/*": ["assets/images/icons*"]
 ```
 
 You can do it in a separate PR if you want as you might need to update `jest.config.js` as well probably!

Reply:
I have tried this but getting error when running the application 😞 I will update it in [another ticket](https://github.com/manh-t/react-survey/issues/60) 🙏 
![image](https://github.com/manh-t/react-survey/assets/60863885/b14934d6-3504-45e3-80a6-a866d6050319)